benchmark/src/scenarios/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/benchmark
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ScenarioFactory} from '../benchmark';
+import {CreateTodo} from './create-todo.scenario';
+import {FindTodos} from './find-todos.scenario';
+
+export interface ScenarioMap {
+  [name: string]: ScenarioFactory;
+}
+
+export const scenarios: ScenarioMap = {
+  'find all todos': FindTodos,
+  // IMPORTANT NOTE(bajtos) the find scenario must run before create.
+  // Otherwise weird data is reported. I was not able to find the cause.
+  'create a new todo': CreateTodo,
```

Comment:
This is weird. When I run "CreateTodo" first and "FindTodos" second, I can see that the sequence is able to handle "findTodo" method in less than 10ms, but yet autocannon is not able to get more than about 10 requests handled for the duration of the benchmark.

I tried may tweaks, from forcing garbage collection to adding delays, but was not able to find the reason of this weird behavior.

I think it's ok to keep this "workaround" for now, we can investigate later if/when we have more time.